src/dogehouse/constants.py

```diff
@@ -9,10 +9,14 @@
 GET_TOP_ROOMS = 'room:get_top'
 ROOMS_FETCHED = 'room:get_top:reply'
 CHAT_DELETE = 'chat:delete'
+CHAT_MEMBER_BAN = 'chat:ban'
```

Comment:
make this one `BAN_CHAT_MEMBER` and same for unban.

I think i should separate out event constants and action constants